docs/source/examples/Detecting Events.mystnb

```diff
@@ -0,0 +1,386 @@
+orbit = Orbit.circular(Earth, 150 * u.km)
+
+# Define a threshold altitude for crossing.
+thresh_alt = 50  # in km
+altitude_cross_event = AltitudeCrossEvent(thresh_alt, R)  # Set up the event.
+events = [altitude_cross_event]
+
+rr, _ = cowell(
+    Earth.k,
+    orbit.r,
+    orbit.v,
+    tofs,
+    events=events,
+    f=f,
+)
+
+print(f"The threshold altitude was crossed {altitude_cross_event.last_t} after the orbit's epoch.")
+```
+
+Let's see how did the orbit's altitude vary with time:
+
+```{code-cell} ipython3
+altitudes = np.apply_along_axis(norm, 1, (rr << u.km).value) - Earth.R.to_value(u.km)
+plt.plot(tofs[:len(rr)].to_value(u.s), altitudes);
+plt.title("Altitude variation")
+plt.ylabel("Altitude (in km)");
+plt.xlabel("Time (in s)");
+```
+Refer to the API documentation of the events to check the default values for `terminal` and `direction` and change it as required.
+
+
+## Latitude Crossing Event
+
+Similar to the `AltitudeCrossEvent`, just pass the threshold latitude while instantiating the event.
+
+```{code-cell} ipython3
+orbit = Orbit.from_classical(
+    Earth, 6900 << u.km, 0.63 * u.one, 30 * u.deg, 0 * u.deg, 0 * u.deg, 0 * u.deg
+)
+```
+
+```{code-cell} ipython3
+thresh_lat = 20 * u.deg
+latitude_cross_event = LatitudeCrossEvent(orbit, thresh_lat, terminal=True)
+events = [latitude_cross_event]
+
+tofs = np.arange(0, 2000, 10) << u.s
+rr, _ = cowell(
+    Earth.k,
+    orbit.r,
+    orbit.v,
+    tofs,
+    events=events,
+)
+print(f"The threshold latitude was crossed {latitude_cross_event.last_t} s after the orbit's epoch")
+```
+
+Let's plot the latitude varying with time:
+
+```{code-cell} ipython3
+latitudes = []
+for r in rr:
+    position_on_body = (r / norm(r)) * Earth.R
+    lat, _, _ = cartesian_to_ellipsoidal(Earth.R, Earth.R_polar, *position_on_body)
+    latitudes.append(np.rad2deg(lat))
+plt.plot(tofs[:len(rr)].to_value(u.s), latitudes);
+plt.title("Latitude variation")
+plt.ylabel("Latitude (in degrees)");
+plt.xlabel("Time (in days)");
+```
+
+The orbit latitude doesn't change after the event was detected since we had set `terminal=True`.
+
+Since the attractor is `Earth`, we could use `GroundtrackPlotter` for showing the groundtrack of the
+orbit on Earth.
+
+```{code-cell} ipython3
+es = EarthSatellite(orbit, None)
+
+# Show the groundtrack plot from
+t_span = time_range(orbit.epoch - 1.5 * u.h, end=orbit.epoch + latitude_cross_event.last_t)
+
+# Generate ground track plotting instance.
+gp = GroundtrackPlotter()
+gp.update_layout(title="Latitude Crossing")
+
+# Plot the above-defined earth satellite.
+gp.plot(
+    es,
+    t_span,
+    label="Orbit",
+    color = "red",
+    marker={"size": 10, "symbol": "triangle-right", "line": {"width": 1, "color": "black"}},
+)
+```
+
+Viewing it in the `orthographic` projection mode,
+
+```{code-cell} ipython3
+gp.update_geos(projection_type="orthographic")
+gp.fig.show()
+```
+
+and voila! The groundtrack terminates almost at the 60 degree latitude mark.
```

Comment:
This line must be updated to the `35 * u.deg`. In fact, it claims to cross latitude at 60 degrees but you used 20 degrees in your example.

Reply:
Oops! There was a typo...